indico/core/notifications.py

```diff
@@ -77,8 +78,13 @@ def _log_email(email, event, module, user, meta=None):
         'body': email['body'].strip(),
         'state': 'pending',
         'sent_dt': None,
+        'attachments': sorted(
+            a.get_filename('unnamed') if isinstance(a, MIMEBase) else a[0]
```

Comment:
In which case is the filename missing? Or is that just something MIMEBase requires?